services/movie.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> None:

```

Comment:
The return type hint for `get_movies` should be `QuerySet[Movie]` instead of `None`, as the function returns a QuerySet containing `Movie` instances.